app/dmc/src/main.c

```diff
@@ -431,8 +431,7 @@ int main(void)
 				}
 
 				if (DT_NODE_HAS_STATUS(DT_ALIAS(fan0), okay)) {
-					pwm_set_cycles(max6639_pwm_dev, 0, UINT32_MAX, UINT32_MAX,
-						       0);
+					pwm_set_cycles(max6639_pwm_dev, 0, UINT8_MAX, UINT8_MAX, 0);
```

Comment:
This seems specific to the implementation / hardware rather than the API. Is it possible to change how the driver handles `UINT32_MAX` / `UINT32_MAX` rather than in the application?